src/web/components/StickyBottomBanner/ReaderRevenueBanner.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { useEffect, useState } from 'react';
+import * as emotion from 'emotion';
+import * as emotionCore from '@emotion/core';
+import * as emotionTheming from 'emotion-theming';
+
+const checkForErrors = (response: any) => {
+    if (!response.ok) {
+        throw Error(
+            response.statusText ||
+                `SlotBanner | An api call returned HTTP status ${response.status}`,
+        );
+    }
+    return response;
+};
+
+const sendOphanEvent = (): void => {
+    // TODO stub
+};
+
+type Props = {
+    isSignedIn?: boolean;
+    countryCode?: string;
+    contentType: string;
+    sectionName?: string;
+    shouldHideReaderRevenue: boolean;
+    isMinuteArticle: boolean;
+    isPaidContent: boolean;
+    isSensitive: boolean;
+    tags: TagType[];
+    contributionsServiceUrl: string;
+};
+
+// TODO specify return type (need to update client to provide this first)
+const buildPayload = (props: Props) => {
+    return {
+        tracking: {
+            // TODO stub
+        },
+        targeting: {
+            ...props,
+            // TODO stub
+        },
+    };
+};
+
+const MemoisedInner = ({
+    isSignedIn,
+    countryCode,
+    contentType,
+    sectionName,
+    shouldHideReaderRevenue,
+    isMinuteArticle,
+    isPaidContent,
+    isSensitive,
+    tags,
+    contributionsServiceUrl,
+}: Props) => {
+    const [Banner, setBanner] = useState<React.FC>();
+    const [bannerProps, setBannerProps] = useState<{}>();
+
+    useEffect(() => {
+        const bannerPayload = buildPayload({
+            isSignedIn,
+            countryCode,
+            contentType,
+            sectionName,
+            shouldHideReaderRevenue,
+            isMinuteArticle,
+            isPaidContent,
+            tags,
+            contributionsServiceUrl,
+            isSensitive,
+        });
+
+        window.guardian.automat = {
```

Comment:
Note more for myself, but we should centralise this somehow to avoid every component having to do it